<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WiFiAnaliza</class>
 <widget class="QMainWindow" name="WiFiAnaliza">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>946</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analiza</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Čvorovi</string>
    </property>
   </widget>
   <widget class="QListView" name="qListViewCvorovi">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelAktivnihCvorova">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Aktivnih čvorova:</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableOkviri">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>631</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Paketi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartStop">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Započni čitanje</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxStatistika">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>1201</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>Statistika</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>67</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Broj paketa po čvoru</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableBrojPaketa">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>40</y>
       <width>471</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
